src/pages/proposals/new-cards.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { FunctionComponent, useEffect, useState } from 'react';
+import { Pagination } from '@material-ui/lab';
+import styled from '@emotion/styled';
+import { RootState } from 'src/reducers/store';
+import { useHistory } from 'react-router-dom';
+import { voteCalculation } from 'src/utils/vote-calculation';
+import TooltipGraphSection from './tooltip-graph-section';
+import { statusColor } from 'src/utils/color-formatter';
+import moment from 'moment';
+
+type CardsProps = {
+	home?: boolean;
+	proposals: any[];
+};
+
+const NewCards: FunctionComponent<CardsProps> = ({ home, proposals }) => {
+	const history = useHistory();
+	const [page, setPage] = useState(1);
+	let rowsPerPage = 15;
+	if (home) {
+		rowsPerPage = 6;
+	}
+
+	const handleClick = (id: string) => {
+		history.push(`/proposals/${id}`);
+	};
+
+	const handleChangePage = (event: React.ChangeEvent<unknown>, page: number) => {
+		setPage(page);
+	};
+
+	const count = Math.ceil(proposals.length / rowsPerPage);
+
+	const formattedItems = proposals.length
+		? proposals.map(function iterateItems(item) {
+				return item.raw;
+		  })
+		: [];
+
+	return (
+		<>
+			<div className="flex flex-wrap">
+				{proposals.length &&
+					proposals.map((proposal, index) => {
+						if (index < page * rowsPerPage && index >= (page - 1) * rowsPerPage) {
+							return (
+								<div onClick={() => handleClick(proposal.id)} key={index} className="w-full md:w-2/4 cursor-pointer">
+									<div className="card bg-card p-4 mb-4 mr-0 md:mr-4 rounded-[20px]">
+										<div className="flex justify-between items-center py-3">
+											<Number>{proposal.id}</Number>
+											<p className={`font-semibold text-transparent bg-clip-text ${statusColor(proposal.status)}`}>
+												{proposal.proposalStatus === 0
```

Comment:
Can we move this to a variable and use a switch statement or an if? This long nested ternary is hard to read